cogs/prometheus/features.py

```diff
@@ -0,0 +1,50 @@
+from prometheus_client import Counter, Gauge
+
+METRIC_PREFIX = "discord_"
+
+COMMANDS_GAUGE = Gauge(METRIC_PREFIX + "stat_total_commands", "Amount of commands")
```

Comment:
Just a nitpick, command is a countable noun...
```suggestion
COMMANDS_GAUGE = Gauge(METRIC_PREFIX + "stat_total_commands", "Number of commands")
```